src/main/java/com/alkemy/ong/repository/ActivityRepository.java

```diff
@@ -2,6 +2,8 @@
 
 import com.alkemy.ong.model.entity.ActivityEntity;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
 
+@Repository //IMPORTANT - Added @Repository annotation in branch OT111-35
```

Comment:
Nota: No hace falta la anotacion ya q eso se va a generar automaticamente cuando se genere la implementación

Igualmente dejo link con una discusión  q tiene mas detalles por si quieres saber

Reply:
Me interesa el link!

Reply:
https://www.it-swarm-es.com/es/java/repository-no-es-necesario-al-implementar-jparepository/832385271/#:~:text=De%20hecho%2C%20no%20es%20necesario,de%20las%20interfaces%20predefinidas%20Repository%20.&text=Esto%20buscar%C3%A1%20Spring%20Data%20repositories,no%20est%C3%A1n%20anotados%20con%20%40Repository%20.